### PR TITLE
Scale seeds

### PR DESCRIPTION
**Pull request by ConConRob**
# Description
-merged from diana's branch to avoid conflicts.
- seeds can now be scaled to large sizes by just changing the company number
- now when the db is seeded there are about 400-500 documents. 
- merged in validation branch

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

#25 

## Change status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

Manually 

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [x] I have commented my code, particularly in hard-to-understand areas

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

### CODE REVIEW

File: data/index.js [hunk 1]
```diff
@@ -4,23 +4,17 @@ const seedChats = require('./seedChats');
...
-  //   }
-  // ];
+  try {
+    await Promise.all([
+      models.Chat.deleteMany({}),
+      models.User.deleteMany({}),
+      models.Hotel.deleteMany({}),
+    ]);
+    let hotels = await seedHotel();
+    let hotelsWithUsers = await seedUsers(hotels);
```
Comment (mark-marshall) [inline data/index.js]: Lets ---> Consts

File: data/seedUsers.js [hunk 2]
```diff
@@ -2,97 +2,83 @@ const faker = require('faker');
...
-      const receptionistId = new mongoose.Types.ObjectId();
-      name = faker.name.firstName();
-      recptionists.push({
-        _id: receptionistId,
+      admins.push({
+        _id: new mongoose.Types.ObjectId(),
+        hotel_id,
+        name: faker.name.firstName(),
+        email: faker.internet.email(),
+        password: '1234',
```
Comment (mark-marshall) [inline data/seedUsers.js]: Passwords will be merged with Pavol's PR including bcrypt

File: data/seedUsers.js [hunk 3]
```diff
@@ -1,98 +1,84 @@
 const faker = require('faker');
 const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
```
Comment (Pav0l) [inline data/seedUsers.js]: should be `bcrytpjs` library

File: data/seedUsers.js [hunk 4]
```diff
@@ -1,98 +1,84 @@
...
-    let name = faker.name.firstName();
-    const superAdminId = new mongoose.Types.ObjectId();
-    hotelIds[i].superAdmin = { superAdminId, name };
+    //make super admin for this hotel
+    const superAdmin = {
+      _id: new mongoose.Types.ObjectId(),
+      hotel_id,
+      name: faker.name.firstName(),
+      email: `superAdmin${i}@superAdmin.com`,
+      password: '1234',
```
Comment (Pav0l) [inline data/seedUsers.js]: please hash these for seeds

File: package.json [hunk 5]
```diff
@@ -17,6 +19,7 @@
   },
   "homepage": "https://github.com/lambdalabs-eu1-webchat/webchat-backend#readme",
   "dependencies": {
+    "bcrypt": "^3.0.6",
```
Comment (Pav0l) [inline package.json]: this library should not be here

File: routes/auth.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const bcrypt = require('bcrypt');
```
Comment (Pav0l) [inline routes/auth.js]: same as above - `bcryptjs`

**Review by Pav0l — CHANGES_REQUESTED** [review]
Great job with data seeding! 🌱

There are couple of comments in code base

**Review by Pav0l — APPROVED** [review]
👍

**Pull request merged**